lib/paymentPage/paymentPage.dart

```diff
@@ -0,0 +1,336 @@
+                                  children: [
+                                    Image(
+                                      image: AssetImage(
+                                          'assets/icons/zm-zanaco-logo-400x400 1.png'),
+                                    ),
+                                    SizedBox(
+                                      width: 20,
+                                    ),
+                                    Text(
+                                      'Zanaco',
+                                      style: TextStyle(
+                                          fontSize: 20,
+                                          fontWeight: FontWeight.bold),
+                                    )
+                                  ],
+                                ),
+                              )),
+                        ]),
+                        selected: false,
+                      ),
+                      Divider(
+                        thickness: 0,
+                        endIndent: 20,
+                        indent: 20,
+                        color: Color.fromARGB(255, 255, 255, 255),
+                      ),
+                      RadioListTile<Payment_methods>(
+                        contentPadding: EdgeInsets.only(left: 40),
+                        value: Payment_methods.fnb,
+                        groupValue: _character,
+                        onChanged: (Payment_methods? value) {
+                          setState(() {
+                            _character = value;
+                          });
+                        },
+                        activeColor: Colors.deepPurple[500],
+                        title: Row(children: <Widget>[
+                          Expanded(
+                              flex: 0,
+                              child: InkWell(
+                                child: Row(
+                                  children: [
+                                    Image(
+                                      image: AssetImage('assets/icons/fnb.png'),
+                                      height: 40,
+                                    ),
+                                    SizedBox(
+                                      width: 40,
+                                    ),
+                                    Text(
+                                      'FNB E-wallet',
+                                      style: TextStyle(
+                                          fontSize: 20,
+                                          fontWeight: FontWeight.bold),
+                                    )
+                                  ],
+                                ),
+                              )),
+                        ]),
+                        selected: false,
+                      ),
+                      SizedBox(
+                        height: 40,
+                      ),
+                      ElevatedButton(
```

Comment:
@batsy3 either use the custom button or refactor this layout and styling into the custom button file under the widgets folder. Rebuilding the button on every page(screen) will be redundant.